apps/src/blockly/customBlocks/googleBlockly/mixins/procedureCallerOnChangeMixin.js

```diff
@@ -39,16 +39,13 @@ const procedureCallerOnChangeMixin = {
     if (!def) {
       // We have no def nor procedure model.
       Blockly.Events.setGroup(event.group);
-      this.model_ = this.createDef_(
-        this.getFieldValue('NAME'),
-        this.paramsFromSerializedState_
-      );
+      this.model_ = this.createDef_(name, this.paramsFromSerializedState_);
```

Comment:
The changes in this file are unrelated to the task, but I noticed there was a `name` variable (declared on line 36, above) that we could use here and below.